app/views/user_mailer/admin_hidden_bookmark_notification.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :message do %>
+  <p><%= t("mailer.general.greeting.formal_html", name: style_bold(@user.login)) %></p>
+
+  <p><%= t(".hidden.html", title: style_creation_link(@bookmark.bookmarkable.title, @bookmark)) %></p>
```

Comment:
And then to match the change in the yml file and also conform to this from Jira:
> In the HTML email, the creation title inside the bookmark link should be bold and italicized.
 
```suggestion
  <p><%= t(".hidden.html", bookmark_of_link: link_to(t(".hidden.bookmark_of", title: tag.em(tag.strong(@bookmark.bookmarkable.title))), @bookmark)) %></p>
```